src/tools/rbd_nbd/rbd-nbd.cc

```diff
@@ -465,9 +467,16 @@ static int open_device(const char* path, bool try_load_module = false)
     if (r < 0) {
       cerr << "rbd-nbd: failed to load nbd kernel module: " << cpp_strerror(-r) << std::endl;
       return r;
+    } else {
+      load_module = true;
     }
     nbd = open(path, O_RDWR);
   }
+
+  if ((nbds_max || set_max_part) && try_load_module && !load_module) {
+    cerr << "rbd-nbd: ignoring kernel module parameter options: nbd module already loaded." << std::endl;
```

Comment:
nit: could you consider breaking the line not to exceed 80 char length? Also the final '.' is not needed. Otherwise lgtm